Specs/Number/German/NumberModel.json

```diff
@@ -756,7 +756,7 @@
   },
   {
     "Input": "1 234 567,89 ist ein gültiges Zahlenformat.",
-    "NotSupported": "javascript",
+    "NotSupported": "javascript,python",
```

Comment:
Do you know why this case is not currenrly supported?

Reply:
This part resolves
`      {
        "Text": "ein",
        "TypeName": "number",
        "Resolution": {
          "value": "1"
        },
        "Start": 17,
        "End": 19
      }`

But this part does not
`
      {
        "Text": "1 234 567,89",
        "TypeName": "number",
        "Resolution": {
          "value": "1234567,89"
        },
        "Start": 0,
        "End": 11
      },
`

Reply:
I see. But why? If this is a port of .NET, it should work.

Reply:
We try to get as close to a "port" as we can. We are happy with the use cases that are working so we did not spend any more time pursuing the ones that are not but we may do again in future.